templates/miq-template.yaml

```diff
@@ -272,7 +272,7 @@ objects:
     annotations:
       description: "Keeps track of changes in the ManageIQ postgresql image"
   spec:
-    dockerImageRepository: docker.io/fbladilo/miq-postgresql
+    dockerImageRepository: docker.io/manageiq/manageiq-pods
```

Comment:
@simaishi I don't think we need to keep track of three different ImageStreams pointing to the same docker repository. We could should use a single "miq-images" IS and reference that across the template. We moved away from three different repos and we now call the specific images for pods by tags, not by image names anymore.